### DISCUSSION TITLE
Invisible Spiders/Charged Creepers causing weird renders to happen

### DISCUSSION THREAD

**Discussion by Turkey9002** [ACGaming/UniversalTweaks#63] [Bugfix Suggestions]
If you play in Hard difficulty you'll end up seeing this really annoying bug a lot due to the invisible spiders that can spawn and for awhile I thought it was caused by something else.
https://bugs.mojang.com/browse/MC-79697

**Comment by ACGaming** [comment]
Implemented in 831b96b79c8ac5e259c36a363169532d73f595ff.
